providers/aws/connection/awsec2ebsconn/provider.go

```diff
@@ -62,14 +62,20 @@ func NewAwsEbsConnection(id uint32, conf *inventory.Config, asset *inventory.Ass
 	if err != nil {
 		return nil, errors.Wrap(err, "could not load instance info: aws-ec2-ebs provider only valid on ec2 instances")
 	}
+	log.Debug().Interface("info", i).Msg("received instance info")
 
 	// ec2 client for the scanner region
 	cfg.Region = i.Region
 	scannerSvc := ec2.NewFromConfig(cfg)
 
+	targetRegion := conf.Options["region"]
+	if targetRegion == "" {
```

Comment:
if confirmed.Options["region"] is empty we actually need to fail out, we can't properly scan unless we know the target's region

Reply:
Oh, I thought I could asume the same region as the scanner.

Reply:
Before, we relied on the `--region` flag. When this wasn't present, the describes error in the issue happened. Now, we still use this flag when present, but fallback to the scanner region.